packages/@react-native-windows/telemetry/src/utils/basePropUtils.ts

```diff
@@ -63,7 +89,9 @@ export function nodeArchitecture(): string {
  * @returns The device platform.
  */
 export function devicePlatform(): string {
-  return platform();
+  const os = platform();
```

Comment:
Can we keep this function behavior as-is and rename it to `nodePlatform()`?

I want to make sure we record what node's `platform()` call returns (since it's our of our control and may be relevant to debugging issues) and would rather stick it in our own common field rather than munge it to fit into the PartA `deviceClass`.

Then we should then have a *different* method to get the correct value for `deviceClass` (which may take the output from `nodePlatform` and transform it to `Windows` for windows PCs, but we should make sure we also put the correct expected values for mac/linux).